ksql-engine/src/test/java/io/confluent/ksql/function/InternalFunctionRegistryTest.java

```diff
@@ -176,4 +181,9 @@ public void shouldAddFunctionWithSameNameClassButDifferentArguments() {
     assertThat(functionRegistry.getUdfFactory("func")
         .getFunction(Collections.emptyList()), equalTo(func));
   }
+
+  @Test(expected = KsqlException.class)
+  public void shouldThrowExceptionIfNoFunctionsWithNameExist() {
```

Comment:
Maybe use ExpectedException to test the error message contains the function name